proto/stream_plan.proto

```diff
@@ -504,7 +504,24 @@ message StreamScanNode {
   uint32 snapshot_read_barrier_interval = 9 [deprecated = true];
 
   // The external table that will be backfilled for CDC.
-  plan_common.ExternalTableDesc cdc_table_desc = 10;
+  plan_common.ExternalTableDesc cdc_table_desc = 10 [deprecated = true];
```

Comment:
Can we completely remove these fields? IIRC, the CDC source table is an experimental feature and not documented.

Reply:
Fair enough, I will remove this field, `StreamScanType::CdcTable`, and the corresponding section in `from_proto::stream_scan.rs`.